src/analytics/service/AnalyticsService.ts

```diff
@@ -17,6 +19,19 @@ export default class AnalyticsService {
     return this.getEmbed(embedCode, timezone);
   }
 
+  public get(): Promise<Report[]> {
+    return this.getReports();
+  }
+
+  private getReports(): Promise<Report[]> {
+    const deferred = Q.defer<Report[]>();
+    const requestUrl = `/${Version.V2}${Resource.ANALYTICS}`;

```

Comment:
@mb-ca this should be GET /v1/analytics since we are trying to re-use the same resource that we had in the old webapp. Correct me if I am wrong @ryankazokas

Reply:
You are right @rjdavis3 . i had the same feedback but never submitted my review.